pixano/datasets/features/schemas/sequence_frame.py

```diff
@@ -57,11 +59,12 @@ def create_sequence_frame(
     image = create_image(item_id, url, id, width, height, format, other_path)
     return SequenceFrame(
         id=image.id,
+        item_id=item_id,
+        sequence_id=sequence_id,
```

Comment:
Is sequence_id really a required field ?

Reply:
It seems to be the case as it is how we link multiple sequence frames to one video.